src/newLaunch/lbp/stages/stage7.html

```diff
@@ -15,25 +15,23 @@
           Prime Launch interface. The approval takes place in the <a href.bind="multiSigWalletUri" target="_blank"
             rel="noopener noreferrer">Prime Launch multisig wallet</a>. Reviews generally take 5-7 days.</p>
 
-        <p>After being approved, the <etherscanlink hide-clipboard-button
-            address.to-view="launchConfig.launchDetails.adminAddress" text="Liquid Launch Admin"></etherscanlink>
-
+        <p>
+          After being approved, the 
+          <etherscanlink hide-clipboard-button address.to-view="launchConfig.launchDetails.adminAddress" text="Liquid Launch Admin"></etherscanlink>
           will have to fund the ${wizardState.launchTypeTitle} with
-
           <formatted-number
             value.to-view="wizardState.requiredProjectTokenDeposit  | ethwei:launchConfig.tokenDetails.projectTokenInfo.decimals"
-            thousands-separated></formatted-number>
-          ${launchConfig.tokenDetails.projectTokenInfo.symbol}
-
-          and
-
+            thousands-separated>
+          </formatted-number>
+          ${launchConfig.tokenDetails.projectTokenInfo.symbol} and 
           <formatted-number
```

Comment:
@hiaux0: Apparently having multiple linebreaks, affect the view. I've removed these extra empty lines (27 & 29), which solved the issue.

All the other changes are just formatting adjustments.